taskManager/src/main/java/com/kunmi/taskManager/taskManager/records/GetProjectResponseRecord.java

```diff
@@ -0,0 +1,8 @@
+package com.kunmi.taskManager.taskManager.records;
+
+import java.util.UUID;
+
+public record GetProjectResponseRecord(
```

Comment:
dto was ok) You dont need use java entity in naming) you already have key word for this. its a same as name class with Class) Like ProjectServiceClass